### DISCUSSION TITLE
Key framing on traditional mode?

### DISCUSSION THREAD

**Discussion by MustardMonday** [keiwando/toonsquid#354] [Q&A]
I know it’s dumb and kinda defeats the purpose of having two different versions, but can you use key frames when having a project set to traditional? I find the timeline much easier than the key frame setting, but I assume you can’t keyfeame on traditional, or if you can, I’m not sure how. Does anyone know if that’s possible? I’m new to this stuff.

**Comment by Cat-Butler** [comment]
Welcome to the discussions!

Yes, it is certainly possible, and I don't think it is at all dumb or defeating the purpose.
The only thing the Traditional/Keyframing switch seems to change is the default number of frames a new drawing automatically occupies, and you can go into the settings and change that afterwards (Actions > Settings > General > Drawings). It does not affect keyframing behaviour or possibilities.

Like you, even for keyframe animation I often prefer to start with either the traditional one frame default, or at least a smaller number than the default 120 frames (5 seconds), and I can then extend the frame length for drawings manually as needed as I work and add keyframes.
If you look at "Saturday morning" animated cartoons with a critical eye, you can see that many of them cut from shot to shot, with each individual shot only taking a few seconds.

Fortunately, ToonSquid is flexible enough to satisfy a variety of work flows and animator preferences.

**Reply by MustardMonday** [reply]
Thank you for the advice! Is it the same as the key frame mode? I can't seem to figure it out.

**Comment by keiwando** [comment]
Yes, both are possible in the same project. Here is the relevant handbook section about the Default Workflow choice when you create a new project:
https://toonsquid.com/handbook/projects/create_project/#workflow

That choice only affects certain default settings of that project. As the handbook mentions:
> All of these settings can be adjusted later on in the editor. Every project is inherently compatible with all workflows - i.e. you can always use traditional animation and keyframing in the same project.
